lib/utils/EventLogRiemannClient.js

```diff
@@ -52,7 +52,7 @@ class EventLogRiemannClient {
 
   handleEvent(message, data) {
     try {
-      if (data.event && _.get(data, 'event.response.status') !== CONST.HTTP_STATUS_CODE.BAD_REQUEST) {
+      if (data.event && !(_.get(config, 'riemann.skip_events_with_client_errors', true) && this.isEventWithClientError(_.get(data, 'event.response.status')))) {
```

Comment:
for the sake of defining constants and having a sample. Can you please add this config param `skip_events_with_client_errors` in settings.yml? Also from readability perspective can this be renamed to `riemann.skip_http_status_40x` - client errors does not immediately strike a bell.

Reply:
Yes , agree . Will make the changes.